src/style.css

```diff
@@ -1 +1,57 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  position: relative;
+  width: 96px;
+  top: 22px;
+  left: 22px;
```

Comment:
You're setting fixed sizes with 'width', 'top', and 'left' properties. If there is no such requirement, consider using sizes that are relative to the content or parent element to prevent overflow or accidental scroll bars.